src/algorithms.jl

```diff
@@ -43,6 +43,7 @@ Anas5(; w=1) = Anas5(w)
 struct OwrenZen3 <: OrdinaryDiffEqAdaptiveAlgorithm end
 struct OwrenZen4 <: OrdinaryDiffEqAdaptiveAlgorithm end
 struct OwrenZen5 <: OrdinaryDiffEqAdaptiveAlgorithm end
+struct LDDRK64 <: OrdinaryDiffEqAdaptiveAlgorithm end
```

Comment:
You accidentally marked it as an adaptive algorithm, which then sets the trait https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/cc3dc9d1f78215c3b125a6fe84b4fc461f7121d3/src/alg_utils.jl#L45 so that way the integrator by default uses adaptive time stepping. Because your method isn't actually adaptive, it thinks every step has infinite error and drops dt to zero, giving your error. To fix this, use `<: OrdinaryDiffEqAlgorithm`

Reply:
Done. But algorithms is still not passing convergence test. Probably there is some mistake in constants or `perform_step!`. I will again take a look by tonight.